src/plugins/expressions/public/mocks.tsx

```diff
@@ -38,10 +36,13 @@ const createStartContract = (): Start => {
     execute: jest.fn(),
     ExpressionLoader: jest.fn(),
     ExpressionRenderHandler: jest.fn(),
-    fork: jest.fn(),
+    getFork: jest.fn(),
```

Comment:
we shouldn't expose public `getFork` function. the creator of fork should expose its fork if it wants to, there should be no other access to it.

Reply:
Fair point. I've removed that 👍